src/mz/src/regions.rs

```diff
@@ -146,17 +124,16 @@ pub(crate) async fn list_regions(
     cloud_providers_and_regions
 }
 
-/**
- * List all the available cloud providers.
- * E.g.: [us-east-1, eu-west-1]
- */
+/// List all the available cloud providers.
+///
+/// E.g.: [us-east-1, eu-west-1]
 pub(crate) async fn list_cloud_providers(
     client: &Client,
     frontegg_auth_machine: &FronteggAuthMachine,
 ) -> Result<Vec<CloudProvider>, Error> {
     let authorization: String = format!("Bearer {}", frontegg_auth_machine.access_token);
 
-    let headers = build_region_request_headers(authorization);
+    let headers = build_region_request_headers(authorization.as_str());
```

Comment:
Should be able to use `&authorization` for these if the funcs are changed to take a `&str`.